stats/view.go

```diff
@@ -43,14 +43,14 @@ type View struct {
 
 	// ss are the channels through which the collected views data for this view
 	// are sent to the consumers of this view.
-	ss map[chan *ViewData]subscription
+	// ss map[chan *ViewData]subscription
```

Comment:
Is the intention to remove this?

Reply:
Yea, will be removed. This is a draft, so I haven't cleaned it up yet.